Iterable/iterable-web-sdk
# More Tests For GetMessages

## JIRA Ticket(s) if any

* [MOB-3149](https://iterable.atlassian.net/browse/MOB-3149)

## Description

Adds unit tests to DOM manipulation-related things for `getMessages()`

## Test Steps

1. Read through the tests and ensure they make sense and are testing the right things and aren't completely pointless
2. Run `yarn test`
3. See full coverage over 90%!!!!!
